### PR TITLE
chore: Update several packages versions and add code of conduct

### PR DESCRIPTION
**Pull request by angel-devo**

### CODE REVIEW

File: devo/common/generic/configuration.py [hunk 1]
```diff
@@ -61,9 +61,7 @@ def load_yaml(self, path, section=None):
         except ImportError as import_error:
             print(
                 str(import_error),
-                "- Use 'pip install pyyaml' or "
-                "install this "
-                "package with [click] option",
+                "- Use 'pip install pyyaml' or " "install this " "package with [click] option",
```
Comment (jgarciai) [inline devo/common/generic/configuration.py:64]: Maybe we can join all strings

File: tests/api/test_query.py [hunk 2]
```diff
@@ -15,13 +15,10 @@ def setUp(self):
             "DEVO_API_QUERY", "from siem.logtrust.web.activity select " "method limit 1"
         )
         self.query_no_results = (
-            'from siem.logtrust.web.activity where method = "OTHER" select'
-            " method limit 1"
+            'from siem.logtrust.web.activity where method = "OTHER" select' " method limit 1"
```
Comment (jgarciai) [inline tests/api/test_query.py:18]: Join strings

File: tests/sender/test_cli.py [hunk 3]
```diff
@@ -163,8 +162,7 @@ def test_cli_notfound_certs(self):
         )
         self.assertIsInstance(result.exception, SystemExit)
         self.assertIn(
-            "Error: Invalid value for '--key': Path "
-            "'not_a_folder/not_a_file' does not exist.",
+            "Error: Invalid value for '--key': Path " "'not_a_folder/not_a_file' does not exist.",
```
Comment (jgarciai) [inline tests/sender/test_cli.py:165]: Join strings

File: tests/sender/test_number_lookup.py [hunk 4]
```diff
@@ -11,9 +11,7 @@ def test_check_is_number(self):
     def test_check_is_not_a_number(self):
         self.assertFalse(
             Lookup.is_number(
-                "5551,HNBId=001D4C-1213120051,"
-                "Fsn=1213120051,bSRName=,"
-                "manualPscUsed=false"
+                "5551,HNBId=001D4C-1213120051," "Fsn=1213120051,bSRName=," "manualPscUsed=false"
```
Comment (jgarciai) [inline tests/sender/test_number_lookup.py:14]: Join strings

File: tests/sender/test_send_data.py [hunk 5]
```diff
@@ -224,51 +224,41 @@ def test_sender_as_handler(self):
...
-            con = Sender.for_logging(
-                config=engine_config, tag=self.my_app, level=TEST_FACILITY
-            )
+            con = Sender.for_logging(config=engine_config, tag=self.my_app, level=TEST_FACILITY)
             logger = get_log(name="DevoLogger", handler=con, level=TEST_FACILITY)
             print("Testing logger info")
-            logger.info(
-                "Testing Sender inherit logging handler functio" "nality... INFO - log"
-            )
+            logger.info("Testing Sender inherit logging handler functio" "nality... INFO - log")
```
Comment (jgarciai) [inline tests/sender/test_send_data.py:230]: Join strings

File: tests/sender/test_send_data.py [hunk 6]
```diff
@@ -224,51 +224,41 @@ def test_sender_as_handler(self):
...
             data_received = TestSender.read(con, 5000)
             print(b"\n" + data_received)
             if len(data_received) == 0:
                 raise Exception("Not msg sent!")
 
             print("Testing logger error")
-            logger.error(
-                "Testing Sender inherit logging handler function" "ality... ERROR - log"
-            )
+            logger.error("Testing Sender inherit logging handler function" "ality... ERROR - log")
```
Comment (jgarciai) [inline tests/sender/test_send_data.py:237]: Join strings

File: tests/sender/test_send_data.py [hunk 7]
```diff
@@ -224,51 +224,41 @@ def test_sender_as_handler(self):
...
             data_received = TestSender.read(con, 5000)
             print(b"\n" + data_received)
             if len(data_received) == 0:
                 raise Exception("Not msg sent!")
 
             print("Testing logger warning")
             logger.warning(
-                "Testing Sender inherit logging handler functio"
-                "nality... WARNING - log"
+                "Testing Sender inherit logging handler functio" "nality... WARNING - log"
```
Comment (jgarciai) [inline tests/sender/test_send_data.py:245]: Join strings

File: tests/sender/test_send_data.py [hunk 8]
```diff
@@ -224,51 +224,41 @@ def test_sender_as_handler(self):
...
             data_received = TestSender.read(con, 5000)
             print(b"\n" + data_received)
             if len(data_received) == 0:
                 raise Exception("Not msg sent!")
 
             print("Testing logger debug")
-            logger.debug(
-                "Testing Sender inherit logging handler functiona" "lity... DEBUG - log"
-            )
+            logger.debug("Testing Sender inherit logging handler functiona" "lity... DEBUG - log")
```
Comment (jgarciai) [inline tests/sender/test_send_data.py:253]: Join strings

File: tests/sender/test_send_data.py [hunk 9]
```diff
@@ -224,51 +224,41 @@ def test_sender_as_handler(self):
...
             data_received = TestSender.read(con, 5000)
             print(b"\n" + data_received)
             if len(data_received) == 0:
                 raise Exception("Not msg sent!")
 
             print("Testing logger critical")
             logger.critical(
-                "Testing Sender inherit logging handler functio"
-                "nality... CRITICAL - log"
+                "Testing Sender inherit logging handler functio" "nality... CRITICAL - log"
```
Comment (jgarciai) [inline tests/sender/test_send_data.py:261]: Join strings

File: tests/sender/test_send_data.py [hunk 10]
```diff
@@ -294,19 +284,15 @@ def test_sender_with_default_logger(self):
...
             )
-            con = Sender.for_logging(
-                config=engine_config, tag=self.my_app, level=TEST_FACILITY
-            )
+            con = Sender.for_logging(config=engine_config, tag=self.my_app, level=TEST_FACILITY)
             # NOTE: this logger logging traces will be visible in console
             con.logger.info(
-                "Testing Sender default handler functionality in "
-                "local console... INFO - log"
+                "Testing Sender default handler functionality in " "local console... INFO - log"
```
Comment (jgarciai) [inline tests/sender/test_send_data.py:290]: Join strings

File: tests/sender/test_send_data.py [hunk 11]
```diff
@@ -294,19 +284,15 @@ def test_sender_with_default_logger(self):
...
             )
             # NOTE: this logger logging traces will be visible in the remote
             # table
             con.info(
-                "Testing Sender default handler functionality in remote "
-                "table... INFO - log"
+                "Testing Sender default handler functionality in remote " "table... INFO - log"
```
Comment (jgarciai) [inline tests/sender/test_send_data.py:295]: Join strings

File: tests/sender/test_send_data.py [hunk 12]
```diff
@@ -347,37 +331,31 @@ def test_sender_as_handler_static(self):
                 raise Exception("Not msg sent!")
 
             print("Testing logger error")
-            logger.error(
-                "Testing Sender static logging handler " "functionality... ERROR - log"
-            )
+            logger.error("Testing Sender static logging handler " "functionality... ERROR - log")
```
Comment (jgarciai) [inline tests/sender/test_send_data.py:334]: Join strings

File: tests/sender/test_send_data.py [hunk 13]
```diff
@@ -347,37 +331,31 @@ def test_sender_as_handler_static(self):
...
             data_received = TestSender.read(con, 5000)
             print(b"\n" + data_received)
             if len(data_received) == 0:
                 raise Exception("Not msg sent!")
 
             print("Testing logger warning")
             logger.warning(
-                "Testing Sender static logging handler "
-                "functionality... WARNING - log"
+                "Testing Sender static logging handler " "functionality... WARNING - log"
```
Comment (jgarciai) [inline tests/sender/test_send_data.py:342]: Join strings

File: tests/sender/test_send_data.py [hunk 14]
```diff
@@ -347,37 +331,31 @@ def test_sender_as_handler_static(self):
...
             data_received = TestSender.read(con, 5000)
             print(b"\n" + data_received)
             if len(data_received) == 0:
                 raise Exception("Not msg sent!")
 
             print("Testing logger debug")
-            logger.debug(
-                "Testing Sender static logging handler " "functionality... DEBUG - log"
-            )
+            logger.debug("Testing Sender static logging handler " "functionality... DEBUG - log")
```
Comment (jgarciai) [inline tests/sender/test_send_data.py:350]: Join strings

File: tests/sender/test_send_data.py [hunk 15]
```diff
@@ -347,37 +331,31 @@ def test_sender_as_handler_static(self):
...
             data_received = TestSender.read(con, 5000)
             print(b"\n" + data_received)
             if len(data_received) == 0:
                 raise Exception("Not msg sent!")
 
             print("Testing logger critical")
             logger.critical(
-                "Testing Sender static logging handler "
-                "functionality... CRITICAL - log"
+                "Testing Sender static logging handler " "functionality... CRITICAL - log"
```
Comment (jgarciai) [inline tests/sender/test_send_data.py:358]: Join strings

File: tests/sender/test_send_lookup.py [hunk 16]
```diff
@@ -236,9 +220,7 @@ def test_check_is_number(self):
     def test_check_is_not_a_number(self):
         self.assertFalse(
             Lookup.is_number(
-                "5551,HNBId=001D4C-1213120051,"
-                "Fsn=1213120051,bSRName=,"
-                "manualPscUsed=false"
+                "5551,HNBId=001D4C-1213120051," "Fsn=1213120051,bSRName=," "manualPscUsed=false"
```
Comment (jgarciai) [inline tests/sender/test_send_lookup.py:223]: Join strings

File: devo/common/generic/configuration.py (hunk 1 above)
Comment (angel-devo) [reply devo/common/generic/configuration.py:64]: OK

File: tests/api/test_query.py (hunk 2 above)
Comment (angel-devo) [reply tests/api/test_query.py:18]: OK

File: tests/sender/test_cli.py (hunk 3 above)
Comment (angel-devo) [reply tests/sender/test_cli.py:165]: OK

File: tests/sender/test_number_lookup.py (hunk 4 above)
Comment (angel-devo) [reply tests/sender/test_number_lookup.py:14]: OK

File: tests/sender/test_send_data.py (hunk 5 above)
Comment (angel-devo) [reply tests/sender/test_send_data.py:230]: OK

**Review by jgarciai — APPROVED** [review]

**Pull request merged**